mne_neuron/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .dipole import simulate_dipole
 from .params import Params
-from .network import Network
+from .network import Network
```

Comment:
missing empty line at end of file. It's what [pep8 recommends](https://www.reddit.com/r/Python/comments/1zjugg/question_why_does_pep8_recommend_leaving_a_blank/)